hitl_install.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Conda environment
+echo "Ensure that you have access to floorplanner hugging face repo before running this script!"
+conda create -y -n siro-hitl-eval python=3.9
+conda activate siro-hitl-eval
+
+# Hab-lab
+git clone --branch stable https://github.com/facebookresearch/habitat-lab.git
+cd habitat-lab
+git checkout 941d4dc
+pip install -e habitat-lab
+pip install -e habitat-baselines
+cd ..
+echo "Step:1/7 Hab-lab SIRo install -- done"
+
+# Hab-sim
+git clone https://github.com/facebookresearch/habitat-sim.git
+cd habitat-sim
+git checkout d32d7510
+python setup.py install --bullet
+echo "Step:2/7 Hab-sim install -- done"
+
+# humanoid data
+cd ../habitat-lab
+wget --no-check-certificate 'https://docs.google.com/uc?export=download&id=1gSNq4oloeX7ZI8mJbr0Kn9sXB5GFDlJc' -O humanoids_skinned.zip
```

Comment:
I hit a weird error with wget and fixed it by updating wget as suggested [here](https://stackoverflow.com/questions/50631185/homebrew-will-not-run-wget-command-library-not-loaded). We could add this tip to a troubleshooting section.

We should probably suggest people run this install script with output redirected to a file, and then they can send us the file for troubleshooting if necessary.